tensorflow/core/kernels/count_up_to_op.cc

```diff
@@ -14,7 +14,8 @@ limitations under the License.
 ==============================================================================*/
 
 #include "tensorflow/core/framework/op_kernel.h"
-#include "tensorflow/core/kernels/variable_ops.h"
+#include "tensorflow/core/framework/resource_mgr.h"
```

Comment:
`/tensorflow/core/kernels/variable_ops.cc:19:10: error: use of private header from outside its module: '/tensorflow/core/framework/op_kernel.h'; 
         ^/tensorflow/core/kernels/variable_ops.cc:20:10: error: use of private header from outside its module: '/tensorflow/core/framework/register_types.h';